hana3d/src/validators/missing_references.py

```diff
@@ -0,0 +1,92 @@
+"""Missing references Validator."""
+import logging
+import os
+from contextlib import suppress
+from typing import List, Set, Tuple
+
+import bpy
+
+from . import BaseValidator, Category
+from ..asset.asset_type import AssetType
+
+
+def _check_missing_reference(image: bpy.types.Image):
+    path = image.filepath_from_user()
+    return os.path.exists(path)
```

Comment:
Acho um pouco confuso a função de `check_missing` retornar `True` quando está presente. Talvez retornar `not os.path.exists(path)` e usar sem o `not` no `if`

Reply:
Acho que prefiro um nome que deixe mais claro ser uma booleana também, algo como `_has_missing_reference`

Reply:
Realmente, se é `missing` deveria retornar false
Vou melhorar o nome!